### PR TITLE
chore: update dependencies

### LINKED ISSUE #216: upgrade to sdk 13

### PR DESCRIPTION
**Pull request by thepiwo**
resolves https://github.com/aeternity/aepp-graffiti/issues/216

### CODE REVIEW

File: aepp/src/utils/walletSearch.js [hunk 1]
```diff
@@ -1,75 +1,97 @@
...
+
+    try {
+      // try to connect to Superhero Wallet
+      aeternity.client = new AeSdkAepp({
+        name: 'AEPP',
+        nodes,
+        onNetworkChange: async (network) => {
+          console.info('onNetworkChange:', network, aeternity.networkId !== network.networkId)
+          await this.aeConnectToNode(network.networkId)
+        }, onAddressChange: async (addresses) => {
```
Comment (kenodressel) [inline aepp/src/utils/walletSearch.js]:
```suggestion
        }, 
        onAddressChange: async (addresses) => {
```

File: aepp/src/utils/walletSearch.js [hunk 2]
```diff
@@ -1,75 +1,97 @@
...
+      })
+
+      await this.scanForWallets()
+
+      if (wallet.walletStatus === 'fallback_static') {
+        aeternity.static = true;
+
+        aeternity.client = new AeSdk({nodes})
+
+        await this.aeConnectToNode(config.defaultNetworkId)
```
Comment (kenodressel) [inline aepp/src/utils/walletSearch.js]:
```suggestion
        aeternity.static = true;
        aeternity.client = new AeSdk({nodes})
        await this.aeConnectToNode(config.defaultNetworkId)
```

File: aepp/src/utils/walletSearch.js [hunk 3]
```diff
@@ -1,75 +1,97 @@
...
+      const handleWallets = async () => {
+        stopScan()
+        resolve(true);
+      }
+
+      const scannerConnection = new BrowserWindowMessageConnection()
+      const stopScan = walletDetector(scannerConnection, handleWallets)
+    }), new Promise((resolve) => setTimeout(() => {
+      resolve(false);
+    }, 1000))]);
```
Comment (kenodressel) [inline aepp/src/utils/walletSearch.js]: maybe longer timeout? I think we initially had 8 seconds

File: aepp/src/views/desktop/Admin.vue [hunk 4]
```diff
@@ -157,15 +157,15 @@
...
   import Util from '~/utils/blockchainUtil'
   import { AeBadge, AeButton, AeIcon, AeLoader } from '@aeternity/aepp-components/src/components'
   import BiggerLoader from '~/components/BiggerLoader'
   import BigNumber from 'bignumber.js'
   import config from '~/config'
   import axios from 'axios'
-  import contractSource from '~/assets/GraffitiAuction.aes'
-  import aeternity from '../../utils/aeternityNetwork'
+  import GraffitiAuctionACI from '../../utils/GraffitiAuctionACI'
+  import {nodes} from '../../utils/aeternityNetwork'
```
Comment (kenodressel) [inline aepp/src/views/desktop/Admin.vue]:
```suggestion
  import { nodes } from '../../utils/aeternityNetwork'
```

File: aepp/src/utils/walletSearch.js (hunk 3 above)
Comment (thepiwo) [reply aepp/src/utils/walletSearch.js]: this seems to work fine with the updated connection logic (copied from governance aepp)

File: aepp/src/views/desktop/Admin.vue (hunk 4 above)
Comment (thepiwo) [reply aepp/src/views/desktop/Admin.vue]: formatting is not very consistent anyways, maybe we can introduce prettier in the future

**Comment by thepiwo** [comment]
@kenodressel everything seems to work now

except for the slot `Swiper` which is swipable only in develop mode, in the production build it will be a list for some reason

**Review by kenodressel — APPROVED** [review]

**Pull request merged**